src/providers/FileSystemProvider/TextSearchProvider.ts

```diff
@@ -29,6 +29,9 @@ export class TextSearchProvider implements vscode.TextSearchProvider {
         regex: query.isRegExp,
         word: query.isWordMatch,
         case: query.isCaseSensitive,
+        // If options.maxResults is null the search is supposed to return an unlimited number of results
+        // Since there's no way for us to pass "unlimited" to the server, I choose a very large number
+        max: options.maxResults !== null ? options.maxResults : 100000,
```

Comment:
```suggestion
        max: options.maxResults ?? 100000,
```

See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Reply:
Cool, I didn't know that